CHANGELOG.md

```diff
@@ -75,6 +75,32 @@ Non-backwards compatible changes
   (highly specific) mode of use. Correspondingly, modules grouped under `*.Categorical.*`
   which exploited these structures for effectful programming have been renamed `*.Effectful`.
 
+#### Moved raw bundles from Data.X.Properties to Data.X.Base
+
+* As mentioned by MatthewDaggitt in Issue #1755, Raw bundles defined in Data.X.Properties
+  should be defined in Data.X.Base as they don't require any properties.
+  * Moved raw bundles From `Data.Nat.Properties` to `Data.Nat.Base`
+  * Moved raw bundles From `Data.Nat.Binary.Properties` to `Data.Nat.Binary.Base`
+  * Moved raw bundles From `Data.Rational.Properties` to `Data.Rational.Base`
+  * Moved raw bundles From `Data.Rational.Unnormalised.Properties` to `Data.Rational.Unnormalised.Base`
+
+#### Moved the definition of RawX from `Algebra.X.Bundles` to `Algebra.X.Bundles.Raw`
```

Comment:
This is not a breaking change right? It can go in the `Major improvements` section :smile: